Sources/Dollar.swift

```diff
@@ -1236,6 +1236,22 @@ open class `$` {
         return array.count
     }
 
+    /// Return the sum of an array of integer numbers
+    ///
+    /// - parameter array: The array to source from.
+    /// - returns: Sum of all int of the array
+    open class func sum(_ array: [Int]) -> Int {
```

Comment:
Use generics instead and you wont have to have separate types defined for int and double.

```swift
open class func sum<T: Numeric>(_ array: [T]) -> T {
```